requirements.txt

```diff
@@ -1 +1 @@
-directory_client_core<1.0.0
+-e .
```

Comment:
this looks weird. Can we make sure we are using pip-tools or pip-env like on other repos?